### PR TITLE
Added option to enable/disable Static Route Advertisement

### PR DESCRIPTION
**Pull request by abdosi**
What I did:

Added option 'advertise':'true'/'false' to STATIC_TALE to control whether given route need to be advertise into BGP.

How I did:

- Add a route in FRR with tag (1 for advertise and 2 for not advertise)
- Create Route Map to match on tag as 1 and permit
- Attach route-map to `redistribute static`.
- Minigraph option to parse `advertise` option of Static Route.

How I verify:

- Unit Test cases
- Manual Verification Device

### CODE REVIEW

**Comment by abdosi** [comment]
@anamehra

File: src/sonic-bgpcfgd/bgpcfgd/managers_static_rt.py [hunk 1]
```diff
@@ -99,44 +102,55 @@ def split_key(key):
...
-            else:
-                op = self.OP_ADD
+        if route_tag != cur_route_tag:
+            for ip_nh in cur_nh_set:
+                op_cmds = op_cmd_list.setdefault(self.OP_DELETE, [])
+                op_cmds.append(self.generate_command(self.OP_DELETE, ip_nh, ip_prefix, vrf, cur_route_tag))
+            for ip_nh in ip_nh_set:
+                op_cmds = op_cmd_list.setdefault(self.OP_ADD, [])
+                op_cmds.append(self.generate_command(self.OP_ADD, ip_nh, ip_prefix, vrf, route_tag))
+        else:
```
Comment (shi-su) [inline src/sonic-bgpcfgd/bgpcfgd/managers_static_rt.py:114]: I was wondering if there are cases where both `route_tag` and `ip_nh_set` change at the same time? If that is the case, would this if-else check skip nh configuration?

Comment (abdosi) [reply src/sonic-bgpcfgd/bgpcfgd/managers_static_rt.py:114]: @shi-su we are deleting all the previous route/nexthop and creating based on new route/nexthop update. This should make sure we use latest nh configuration.

**Review by anamehra — APPROVED** [review]

**Review by shi-su — APPROVED** [review]

**Pull request merged**